fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/api/registry/FlammableBlockRegistry.java

```diff
@@ -21,15 +21,19 @@
 import net.minecraft.tag.Tag;
 
 import net.fabricmc.fabric.api.util.Block2ObjectMap;
-import net.fabricmc.fabric.impl.content.registry.FlammableBlockRegistryImpl;
+import net.fabricmc.fabric.impl.content.registry.LegacyFlammableBlockRegistryImpl;
 
+/**
+ * @deprecated replaced by {@link net.fabricmc.fabric.api.content.registry.v1.FlammableBlockRegistry}
+ */
+@Deprecated
 public interface FlammableBlockRegistry extends Block2ObjectMap<FlammableBlockRegistry.Entry> {
 	static FlammableBlockRegistry getDefaultInstance() {
 		return getInstance(Blocks.FIRE);
 	}
 
 	static FlammableBlockRegistry getInstance(Block block) {
-		return FlammableBlockRegistryImpl.getInstance(block);
+		return LegacyFlammableBlockRegistryImpl.getInstance(block);
```

Comment:
Is there a way we could inline the old api to call new api/impl?

Reply:
See https://github.com/FabricMC/fabric/pull/576/files#r408796138